contracts/SuiBridge.sol

```diff
@@ -52,14 +53,15 @@ contract SuiBridge is ISuiBridge, CommitteeUpgradeable, PausableUpgradeable {
         weth9 = IWETH9(_weth9);
 
         for (uint8 i = 0; i < _supportedChainIDs.length; i++) {
-            // require(_supportedChainIDs[i] != BridgeCommittee.chainID, "SuiBridge: Cannot support self");
             isChainSupported[_supportedChainIDs[i]] = true;
         }
     }
 
     /* ========== EXTERNAL FUNCTIONS ========== */
 
-    /// @dev Transfers tokens with signatures.
+    /// @notice Transfers tokens to the target chain using the provided signatures and message.
```

Comment:
Good callout. Yes I think `claimBridgedTokensWithSignatures` is more appropriate. I'll update the documentation too.

Reply:
Thinking on this more... "claim" suggests that the bridged tokens will be transferred to the caller, when the recipient of the bridged tokens is declared in the message... and its possible the caller of this function won't actually receive the tokens (example, I want to transfer bridged tokens to a new wallet, but execute the tx from another wallet that already has funds).

Reply:
`claimBridgedTokensForReceiver`?

Reply:
all of the bridge committee controlled functions have a "WithSignature" naming convention so it would be `claimBridgedTokensForReceiverWithSignatures` feels overly verbose. I'd prefer `claimBridgedTokensWithSignatures`. What about `transferBridgedTokensWithSignatures`?

Reply:
well now we are back to the original point lol. No strong opinions from me, plz make a call :)

Reply:
🫡